homeassistant/components/media_player/plex.py

```diff
@@ -189,6 +199,18 @@ def update_devices():
         if new_plex_clients:
             add_devices_callback(new_plex_clients)
 
+        # Figure out if device is unavailable and if it should be removed
+        for cid in hass.data[PLEX_DATA].keys():
+            cclient = plex_clients[cid]
+            config_purge_int = config.get(CONF_CLIENT_PURGE_INTERVAL)
+            cclient.set_availability(cid in available_ids)
+            if config.get(CONF_PURGE_UNAVAILABLE_CLIENTS)\
+                    and cclient.unavailable_time is not None \
+                    and cclient.marked_for_purge is not True \
+                    and cclient.unavailable_time.seconds > config_purge_int:
```

Comment:
If `unavailable_time` is a timedelta, use `total_seconds()`

Reply:
Will do, but what is the benefit of doing it that way? or is it purely stylistic to keep with the project norms?